pkg/images/images_test.go

```diff
@@ -130,17 +130,17 @@ func Test_toKeep(t *testing.T) {
 			name: "2.7.0 < min < max < 2.9.0",
```

Comment:
need to update name here to 
```suggestion
			name: "2.7.0 < min < max < 2.10.0",
```